packages/manager/modules/account-sidebar/package.json

```
@@ -10,10 +10,26 @@
   },
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
-  "main": "src/index.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/welcome.js",
```

Pointing to wrong dist here
```suggestion
  "browser": "./dist/umd/account-sidebar.js",
``` maybe

---

copy paste :/ thanks